### PR TITLE
PLD: add travis yml

### PR DESCRIPTION
**Pull request by pld**

### CODE REVIEW

**Comment by prabhasp** [comment]
@pld should this branch be passing?

**Comment by pld** [comment]
no, this doesn't address fixing the tests, just building them

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+lein: lein2
+language: clojure
+script:
+    - lein2 midje
```
Comment (royrutto) [inline .travis.yml]: midje may not be required.

**Comment by royrutto** [comment]
@prabhasp @pld will merge this in and use it in the `test` branch

**Pull request merged**